WorkloadManagementSystem/Utilities/PilotCStoJSONSynchronizer.py

```diff
@@ -335,8 +346,10 @@ def _syncScripts(self):
       tarPath = os.path.join(self.workDir, 'pilot.tar')
       with tarfile.TarFile(name=tarPath, mode='w') as tf:
         for ptf in tarFiles:
+          # This copy makes sure that all the files in the tarball are accessible
+          # in the work directory. It should be kept
```

Comment:
The original idea was that all the files that we are uploading are actually in one place, such that they could be synced somewhere else (with `rsync`, `scp`, `cron`, you name it). So for that, we needed to have them in a "clean" directory, since for the time being, they were wherever the process was being executed. 
So in order to preserve the existing behaviour and also allow for external sync, we :
* work within a `workdir` 
* still upload to the dirac web app
* let the files in place, such that they can be synced by an external tool

Note that there are still flaws, since the list of servers in `pilotFileServer` are used both by the pilots to download and by the CSToJSON to upload. But in fact, the upload will only work on a DIRAC server, so you have plenty of errors in the logs.... That's why I added this flag to disable the upload altogether